livedata/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -484,7 +484,7 @@ class ChatDomainImpl private constructor(
         }
 
     private fun queryEvents(cids: List<String>): List<ChatEvent> {
-        val response = client.getSyncHistory(cids, syncState?.lastSyncedAt ?: NEVER).execute()
+        val response = client.getSyncHistory(cids, syncState?.lastSyncedAt ?: Date()).execute()
```

Comment:
If l`astSyncAt` is null should we pass the current timestamp? This is a value that's narrowing the results, right?

Reply:
Never doesn't work since you hit the limit of how far back you can go and the request is ignored. @tbarbugli what do you think is the right default?